README.md

```diff
@@ -781,6 +792,36 @@ Please select how to proceed:
     Skip this file upload
 ```
 
+### Understanding Bundled Transactions vs V2 Transactions<a id="bundle-vs-v2"></a>
```

Comment:
I think it could be beneficial to reduce the amount of information conveyed in these sections to get the following key points across to the reader quickly:
```
• Bundling saves AR and enhances ArFS reliability by sending associated ArFS transactions up as one atomic bundle.
• Bundled transactions are treated as a single data transaction by the Arweave network, but can be presented as separate transactions by the Arweave Gateway once they have been "unbundled".
• Unbundling can take anywhere from a few minutes up to an hour. During that time, the files in the bundle will neither appear in list- commands nor be downloadable. Similarly, they will not appear in the web app after syncs until unbundling is complete. This can negatively affect the accuracy of upsert operations, so it's best to wait before retrying bulk uploads.
• Bundling is the default option for the CLI, but can be disabled by using the --no-bundle option.
• Bundling reliability on the gateway side degrades once bundles reach either 500 data items (or ~250 files) or 500MiB, so the CLI will create and upload multiple bundles as necessary, or will send files that are simply too large for reliable bundling as unbundled txs.
```
Many of the points I've listed there are included in your prose below. But if we can get straight to the point without too much supplementary info, I think it could streamline uptake for the user. For example, I don't think it's critically important for users to know about V2 transactions, particularly by that proper name, unless it's a term we need to use often to convey how to use the CLI correctly.

Reply:
Sounds good, I've added your suggestion here: 684fe92

Let me know if I've taken too much away